routers/routes/routes.go

```diff
@@ -523,6 +523,10 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+
+		m.Group("/branches", func() {
+			m.Get("/_new/*", context.RepoRef(), repo.CreateBranch)
```

Comment:
Why `_new` instead of `new` ? (Except that we'd need to add another reserved name)